dcs/triggers.py

```diff
@@ -48,15 +55,22 @@ def load_from_dict(self, data):
                 mapping.Point(imp_zone["x"], imp_zone["y"]),
                 imp_zone["radius"],
                 imp_zone["hidden"],
-                imp_zone["name"]
+                imp_zone["name"],
+                imp_zone["color"],
+                imp_zone.get("properties",{})
             )
-            tz.color = imp_zone["color"]
             self._zones.append(tz)
             self.current_zone_id = max(self.current_zone_id, tz.id)
 
-    def add_triggerzone(self, position: mapping.Point, radius=1500, hidden=False, name="") -> TriggerZone:
+    def add_triggerzone(self, position: mapping.Point, radius=1500, hidden=False, name="", color=None, properites={}) -> TriggerZone:
```

Comment:
Looks like I spelled `properties` wrong here.

Reply:
@rp- , does this also need to be `properties=None` and follow the same pattern as what I had done with color in `__init__`?

Reply:
yes same problem